sources/code/main.ts

```diff
@@ -96,7 +96,11 @@ const chromiumVersion: string = process.versions.chrome;
  * if you're improving the code of this application
  */
 
-let appContributors: Array<string> = [appAuthor];
+ //console.log(packageJson)
+ //console.log(packageJson.contributors)
+
+//let appContributors: Array<string> = [appAuthor + packageJson.contrib.authors];
+let appContributors: Array<string> = [ appAuthor, ...packageJson.contributors ];
```

Comment:
Ok, I resolved this part myself, now it at least runs...

Also, because I plan to *squash and merge* everything, I can stand making junk comments to this (`oxmc:patch-3`) repository instead of creating new PR or fast-forward (`git push -f`) pushes (as everything will be merged into one commit anyway)...